lms/djangoapps/teams/static/teams/js/spec/views/team_profile_spec.js

```diff
@@ -59,7 +59,8 @@ define([
                 requests,
                 'GET',
                 interpolate(
-                    '/courses/%(courseID)s/discussion/forum/%(topicID)s/inline?page=1&ajax=1',
+                    '/courses/%(courseID)s/discussion/forum/%(topicID)s/inline
+                    ?page=1&sort_key=activity&sort_order=desc&ajax=1',
```

Comment:
GitHub doesn't seem to like this. I would use concatenation instead of a string spanning two lines, if that works.